ipn/ipnlocal/local.go

```diff
@@ -532,7 +536,14 @@ func NewLocalBackend(logf logger.Logf, logID logid.PublicID, sys *tsd.System, lo
 		captiveCancel:         nil, // so that we start checkCaptivePortalLoop when Running
 		needsCaptiveDetection: make(chan bool),
 	}
-	nb := newNodeBackend(ctx, b.sys.Bus.Get())
+	if sys != nil {
```

Comment:
I think this cannot ever be false, as reaching here we have already gone through places that would panic if it were.